user snippets/elementRightClick.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Simulates right-clicking an element to display a context menu.
+ * Customize by adding a CSS selector to the element variable.
+ * To use an xPath, integrate the getElementByXpath snippet.
+ * 
+ * @param {object} - mablInputs Object containing input
+ *                   variables (mablInputs.variables.user)
+ * @param {function} callback - The callback function
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+    let element = document.querySelector(/*cssSelector*/);
+    
+    let e = element.ownerDocument.createEvent('MouseEvents');
+    e.initMouseEvent('contextmenu', true, true,
+        element.ownerDocument.defaultView, 1, 0, 0, 0, 0, false,
+        false, false, false, 2, null);
```

Comment:
might be a little more clear to set some of these things in variables, so the user can understand what the 1's and 0's and booleans mean in the `initMouseEvent` function. Are all of these inputs to `initMouseEvent` necessary, or are some of them optional?